drivers/virtualbox/virtualbox.go

```diff
@@ -77,9 +77,9 @@ func GetCreateFlags() []cli.Flag {
 			Value:  20000,
 		},
 		cli.StringFlag{
-			EnvVar: "VIRTUALBOX_BOOT2DOCKER_URL",
-			Name:   "virtualbox-boot2docker-url",
-			Usage:  "The URL of the boot2docker image. Defaults to the latest available version",
+			EnvVar: "VIRTUALBOX_ISO",
```

Comment:
This also needs to be updated in the "de-marshalling" bit which translate flags into struct parameters in the driver's `SetConfigFromFlags` method.